addons/resource.language.en_gb/resources/strings.po

```diff
@@ -16515,7 +16524,7 @@ msgstr ""
 #. Description of setting "Videos -> Library -> Show All Items entry"
 #: system/settings/settings.xml
 msgctxt "#38012"
-msgid "Show \"All items\" entry in directory (for example \"All albums\" or \"All seasons\")"
+msgid "Show \"All items\" entry in directory (for example, \"All albums\" or \"All seasons\")"
```

Comment:
I dont mind using e.g. but a few e.g. entries start after a full stop and whoever bastardizes this to E.g. should be shot.

I will chnage to e.g. soon as a proper solution of not using E.g. is offered.